src/actions/profile-view.js

```diff
@@ -874,7 +870,7 @@ export function changeExpandedCallNodes(
 
 export function changeSelectedMarker(
   threadIndex: ThreadIndex,
-  selectedMarker: IndexIntoRawMarkerTable | null
+  selectedMarker: MarkerIndex | null
```

Comment:
There was several instances of this error.
Well error... everything is a number anyway.